onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -556,6 +556,7 @@ def sample_inputs_bernoulli_p_deterministic(op_info, device, dtype, requires_gra
 OP_DB: List[opinfo_core.OpInfo] = [
     opinfo_core.OpInfo(
         "aten._local_scalar_dense",
+        # pylint: disable=protected-access
         op=torch.ops.aten._local_scalar_dense,
```

Comment:
```suggestion
        op=torch.ops.aten._local_scalar_dense,  # pylint: disable=protected-access
```

Otherwise it will disable the rule for everything below this line

Reply:
Also possible:

`# pylint: disable-next=protected-access`

https://stackoverflow.com/questions/28829236/is-it-possible-to-ignore-one-single-specific-line-with-pylint